app/controllers/concerns/can_resume_call_recording.rb

```diff
@@ -4,19 +4,24 @@ module CanResumeCallRecording
   extend ActiveSupport::Concern
 
   def check_and_resume_call_recording
-    return unless WasteCarriersEngine::FeatureToggle.active?(:control_call_recording)
+    return unless should_resume_call_recording?
 
-    flash[:call_recording] = if call_recording_service.resume
-                               { success: t("shared.call_recording_banner.call_resuming.success") }
-                             else
-                               { error: t("shared.call_recording_banner.call_resuming.error") }
-                             end
+    resume_call_recording
   end
 
   private
 
+  def should_resume_call_recording?
+    WasteCarriersEngine::FeatureToggle.active?(:control_call_recording) &&
+      (!@transient_registration.lower_tier? || @transient_registration.is_a?(OrderCopyCardsRegistration))
```

Comment:
If it's a copy cards order the registration must be upper tier - why check both?